src/app/question/question.component.html

```diff
@@ -22,12 +22,15 @@
                 <textarea
                   mdInput
                   mdTextareaAutosize
+                  #question
                   minRows="3"
                   name="inp-question"
+                  maxlength="1000"
                   [(ngModel)]="currentQuestion"
                   placeholder="Write your question here..."
                   formControlName="search"
                 ></textarea>
+                <md-hint align="end" *ngIf="question.value.length > 0">{{ question.value.length }} / 1000</md-hint>
```

Comment:
@binarious just noticed that hint labels are not displayed at all (even on the current build in the chat). Probably this is because we changed from input to textarea..